pkg/systemd/quadlet/quadlet.go

```diff
@@ -2249,3 +2272,33 @@ func addVolumes(quadletUnitFile, serviceUnitFile *parser.UnitFile, groupName str
 
 	return nil
 }
+
+// Add hosts to the podman command
+// An IP can refer to one or multiple hosts following the format :
+// my-host-name;my-second-host-name;third-hostname:192.168.10.10
+func addHosts(quadletUnitFile *parser.UnitFile, groupName string, podman *PodmanCmdline) error {
+	addHosts := quadletUnitFile.LookupAll(groupName, KeyAddHost)
+	for _, addHost := range addHosts {
+		parts := strings.SplitN(addHost, ":", 2)
+
+		if len(parts) == 2 {
+			hostsList := parts[0]
+			ip := parts[1]
+
+			if len(ip) > 0 {
```

Comment:
I think there should also be an `else` here with an error, no?

Reply:
I've added it and changed the error message to log the full line when IP or host is empty.